src/common/model/settings.ts

```diff
@@ -12,4 +12,16 @@ export const settingsSchema = Type.Object({
     battlesHidePvE: Type.Boolean({ default: false }),
     battlesHideLocked: Type.Boolean({ default: false }),
     battlesHideEmpty: Type.Boolean({ default: true }),
+    queue1v1: Type.Boolean({ default: true }),
+    queue2v2: Type.Boolean({ default: true }),
+    queue3v3: Type.Boolean({ default: true }),
+    queue4v4: Type.Boolean({ default: true }),
+    queue5v5: Type.Boolean({ default: true }),
+    queue6v6: Type.Boolean({ default: true }),
+    queue7v7: Type.Boolean({ default: true }),
+    queue8v8: Type.Boolean({ default: true }),
+    queueSmallFFA: Type.Boolean({ default: true }),
+    queueMediumFFA: Type.Boolean({ default: true }),
+    queueLargeFFA: Type.Boolean({ default: true }),
+    queueTeamFFA: Type.Boolean({ default: true }),
```

Comment:
rather than having a separate setting for each queue, I think it would be better to store the enabled queues as an array of the queue ids, like `enabledQueues: Type.Array(Type.String(), { default: [] })`